### PR TITLE
CPU feature copy-and-build from pqclean

### PR DESCRIPTION
**Pull request by baentsch**
Together with PR #654 this PR will address issue #624. CI is expected to fail as `copy_from_pqclean` is not run as regular part of the build pipeline. This PR is meant to allow review of the generator scripts. A full PR can be created after running `copy_from_pqclean` (with many more changes caused by the script).

### CODE REVIEW

File: scripts/copy_from_pqclean/copy_from_pqclean.py [hunk 1]
```diff
@@ -23,12 +31,11 @@ def file_put_contents(filename, s, encoding=None):
 def generator(destination_filename, template_filename, family, scheme_desired):
     template = file_get_contents(os.path.join('scripts', 'copy_from_pqclean', template_filename))
     f = copy.deepcopy(family)
+    #if (destination_filename == "src/kem/kyber/kem_kyber_512.c"):
```
Comment (dstebila) [inline scripts/copy_from_pqclean/copy_from_pqclean.py]: Is this line still needed?

Comment (baentsch) [reply scripts/copy_from_pqclean/copy_from_pqclean.py]: Absolutely no. Removed and merged with upstream.

**Comment by dstebila** [comment]
The directory renaming from hyphens to underscores is leading to errors when running CMake:

```bash
cmake -GNinja ..
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/dstebila/Desktop/OQS/liboqs/build/cpu_features-download
[1/6] Performing update step for 'cpu_features'
[2/6] Performing configure step for 'cpu_features'
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/dstebila/Desktop/OQS/liboqs/build/cpu_features-build
[3/6] Performing build step for 'cpu_features'
ninja: no work to do.
[4/6] No install step for 'cpu_features'
[5/6] No test step for 'cpu_features'
[6/6] Completed 'cpu_features'
-- Configuring done
CMake Error at src/sig/falcon/CMakeLists.txt:25 (add_library):
  Cannot find source file:

    pqclean_falcon-512_clean/common.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/falcon/CMakeLists.txt:32 (add_library):
  Cannot find source file:

    pqclean_falcon-1024_clean/common.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/mqdss/CMakeLists.txt:25 (add_library):
  Cannot find source file:

    pqclean_mqdss-48_clean/sign.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/mqdss/CMakeLists.txt:32 (add_library):
  Cannot find source file:

    pqclean_mqdss-64_clean/sign.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:88 (add_library):
  Cannot find source file:

    pqclean_rainbowIa-classic_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:95 (add_library):
  Cannot find source file:

    pqclean_rainbowIa-cyclic_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:102 (add_library):
  Cannot find source file:

    pqclean_rainbowIa-cyclic-compressed_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:109 (add_library):
  Cannot find source file:

    pqclean_rainbowIIIc-classic_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:116 (add_library):
  Cannot find source file:

    pqclean_rainbowIIIc-cyclic_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:123 (add_library):
  Cannot find source file:

    pqclean_rainbowIIIc-cyclic-compressed_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:130 (add_library):
  Cannot find source file:

    pqclean_rainbowVc-classic_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:137 (add_library):
  Cannot find source file:

    pqclean_rainbowVc-cyclic_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/rainbow/CMakeLists.txt:144 (add_library):
  Cannot find source file:

    pqclean_rainbowVc-cyclic-compressed_clean/rainbow_keypair.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:331 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-128f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:338 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-128f-simple_clean/thash_haraka_simple.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:345 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-128s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:352 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-128s-simple_clean/thash_haraka_simple.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:359 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-192f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:366 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-192f-simple_clean/thash_haraka_simple.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:373 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-192s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:380 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-192s-simple_clean/thash_haraka_simple.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:387 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-256f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:394 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-256f-simple_clean/thash_haraka_simple.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:401 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-256s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:408 (add_library):
  Cannot find source file:

    pqclean_sphincs-haraka-256s-simple_clean/thash_haraka_simple.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:415 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-128f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:422 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-128f-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:429 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-128s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:436 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-128s-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:443 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-192f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:450 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-192f-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:457 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-192s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:464 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-192s-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:471 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-256f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:478 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-256f-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:485 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-256s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:492 (add_library):
  Cannot find source file:

    pqclean_sphincs-sha256-256s-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:499 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-128f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:506 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-128f-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:513 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-128s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:520 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-128s-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:527 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-192f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:534 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-192f-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:541 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-192s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:548 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-192s-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:555 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-256f-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:562 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-256f-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:569 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-256s-robust_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/sphincs/CMakeLists.txt:576 (add_library):
  Cannot find source file:

    pqclean_sphincs-shake256-256s-simple_clean/fors.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx


CMake Error at src/sig/falcon/CMakeLists.txt:32 (add_library):
  No SOURCES given to target: falcon_1024_clean


CMake Error at src/sig/falcon/CMakeLists.txt:25 (add_library):
  No SOURCES given to target: falcon_512_clean


CMake Error at src/sig/mqdss/CMakeLists.txt:32 (add_library):
  No SOURCES given to target: mqdss_31_64_clean


CMake Error at src/sig/mqdss/CMakeLists.txt:25 (add_library):
  No SOURCES given to target: mqdss_31_48_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:102 (add_library):
  No SOURCES given to target: rainbow_Ia_cyclic_compressed_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:137 (add_library):
  No SOURCES given to target: rainbow_Vc_cyclic_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:144 (add_library):
  No SOURCES given to target: rainbow_Vc_cyclic_compressed_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:116 (add_library):
  No SOURCES given to target: rainbow_IIIc_cyclic_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:109 (add_library):
  No SOURCES given to target: rainbow_IIIc_classic_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:130 (add_library):
  No SOURCES given to target: rainbow_Vc_classic_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:123 (add_library):
  No SOURCES given to target: rainbow_IIIc_cyclic_compressed_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:95 (add_library):
  No SOURCES given to target: rainbow_Ia_cyclic_clean


CMake Error at src/sig/rainbow/CMakeLists.txt:88 (add_library):
  No SOURCES given to target: rainbow_Ia_classic_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:569 (add_library):
  No SOURCES given to target: sphincs_shake256_256s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:562 (add_library):
  No SOURCES given to target: sphincs_shake256_256f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:555 (add_library):
  No SOURCES given to target: sphincs_shake256_256f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:548 (add_library):
  No SOURCES given to target: sphincs_shake256_192s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:527 (add_library):
  No SOURCES given to target: sphincs_shake256_192f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:520 (add_library):
  No SOURCES given to target: sphincs_shake256_128s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:513 (add_library):
  No SOURCES given to target: sphincs_shake256_128s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:506 (add_library):
  No SOURCES given to target: sphincs_shake256_128f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:499 (add_library):
  No SOURCES given to target: sphincs_shake256_128f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:457 (add_library):
  No SOURCES given to target: sphincs_sha256_192s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:485 (add_library):
  No SOURCES given to target: sphincs_sha256_256s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:534 (add_library):
  No SOURCES given to target: sphincs_shake256_192f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:478 (add_library):
  No SOURCES given to target: sphincs_sha256_256f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:576 (add_library):
  No SOURCES given to target: sphincs_shake256_256s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:429 (add_library):
  No SOURCES given to target: sphincs_sha256_128s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:464 (add_library):
  No SOURCES given to target: sphincs_sha256_192s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:450 (add_library):
  No SOURCES given to target: sphincs_sha256_192f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:366 (add_library):
  No SOURCES given to target: sphincs_haraka_192f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:401 (add_library):
  No SOURCES given to target: sphincs_haraka_256s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:436 (add_library):
  No SOURCES given to target: sphincs_sha256_128s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:422 (add_library):
  No SOURCES given to target: sphincs_sha256_128f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:387 (add_library):
  No SOURCES given to target: sphincs_haraka_256f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:408 (add_library):
  No SOURCES given to target: sphincs_haraka_256s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:492 (add_library):
  No SOURCES given to target: sphincs_sha256_256s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:345 (add_library):
  No SOURCES given to target: sphincs_haraka_128s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:373 (add_library):
  No SOURCES given to target: sphincs_haraka_192s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:394 (add_library):
  No SOURCES given to target: sphincs_haraka_256f_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:359 (add_library):
  No SOURCES given to target: sphincs_haraka_192f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:541 (add_library):
  No SOURCES given to target: sphincs_shake256_192s_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:415 (add_library):
  No SOURCES given to target: sphincs_sha256_128f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:331 (add_library):
  No SOURCES given to target: sphincs_haraka_128f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:352 (add_library):
  No SOURCES given to target: sphincs_haraka_128s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:471 (add_library):
  No SOURCES given to target: sphincs_sha256_256f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:380 (add_library):
  No SOURCES given to target: sphincs_haraka_192s_simple_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:443 (add_library):
  No SOURCES given to target: sphincs_sha256_192f_robust_clean


CMake Error at src/sig/sphincs/CMakeLists.txt:338 (add_library):
  No SOURCES given to target: sphincs_haraka_128f_simple_clean


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Comment by baentsch** [comment]
Superseded by PR #686

**Pull request closed without merging**